### PR TITLE
updated dispatch

### PR DESCRIPTION
**Pull request by ghost**
## What problem does this PR solve?

update to python 3

## How did you fix the problem?

fixed dispatch admin

### CODE REVIEW

**Comment by Rowansdabomb** [comment]
I am a bit unclear as to what this PR is doing... Where is `.idea/` directory? I do not see it in the develop branch, also why are you deleting it?

**Comment by Rowansdabomb** [comment]
The title of this PR is ambiguous. Also, the description says this will update dispatch to python 3, but there are no changes to any python versioning that I can see.

File: dispatch/bin/dispatch-admin [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```
Comment (Rowansdabomb) [inline dispatch/bin/dispatch-admin:1]: This is a comment is it not? does this actually cause any sort of change to the function of the code?

Comment (razvannesiu) [reply dispatch/bin/dispatch-admin:1]: It is not a comment, even though it starts with a #. It is called a [shebang line](https://stackoverflow.com/questions/2429511/why-do-people-write-the-usr-bin-env-python-shebang-on-the-first-line-of-a-pyt). When you execute dispatch-admin, that line determines what interpreter will run this code.

**Comment by ghost** [comment]
We are going to close this PR and open a new one with all the files we have modified for the update. All the files from .idea had conflicts with develop branch so we removed them. Those are created by PyCharm.

**Pull request closed without merging**